src/helpers/getPackagesPaths.js

```diff
@@ -0,0 +1,19 @@
+import globby from 'globby'
+import { join, sep } from 'path'
+
+export function getPackagesPaths(cwd, glob = []) {
+  return globby
```

Comment:
I think `packagesPath` should be wrapped in a function so that passing an invalid `packageName` does not return `undefined` but throws an error. This was my initial idea with a function (not showing in tests where I used mocked it). Using `globby` is a really good idea but we should really not have invalid paths passed around (it can get really nasty).